src/routes/DigitalAssets/routes/Send/index.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import DigitalAssetsSendViewContainer from './DigitalAssetsSendViewContainer'
+
+const send = {
+  path: 'send',
+  component: DigitalAssetsSendViewContainer,
+}
+
+const sendAsset = {
+  path: 'send/:asset/',
+  component: DigitalAssetsSendViewContainer,
+}
+
+const sendAssetTo = {
+  path: 'send/:asset/:to',
+  component: DigitalAssetsSendViewContainer,
+}
+
+const repeatPayment = {
+  path: 'send/repaat/:txhash',
+  component: DigitalAssetsSendViewContainer,
+}
+
+export default {
+  send,
+  sendAsset,
+  sendAssetTo,
+  repeatPayment,
+}
```

Comment:
we need only one route
and parse URL query to fill some fields, if their values are valid